lib/flatpak-installation.c

```diff
@@ -1092,6 +1092,68 @@ flatpak_installation_remove_remote (FlatpakInstallation *self,
 }
 
 /**
+ * flatpak_installation_set_config_sync:
+ * @self: a #FlatpakInstallation
+ * @key: the name of the key to set
+ * @value: the new value, or %NULL to unset
+ * @cancellable: (nullable): a #GCancellable
+ * @error: return location for a #GError
+ *
+ * Set a global configuration option for the installation, currently
+ * the only supported key is "languages", which is a comman-separated
+ * list of langue codes like "sv;en;pl", or "" to mean all languages.
```

Comment:
Hmm... are you sure? My understanding of the current code was that presently the languages setting is a list of subpaths like /sv;/en;/pl, unless a lower layer is more forgiving and doesn't require absolute paths. I'm tweaking around with that code in my branch which aims to add the default languages to deploys when upgrading, but I could be wrong.

Reply:
Yes, this changed recently when i did the flatpak config stuff (although it still handles existing configs with slashes.